drivers/gpio/gpio_cc32xx.c

```diff
@@ -126,6 +149,105 @@ static inline int gpio_cc32xx_read(struct device *port,
 	return 0;
 }
 
+static int gpio_cc32xx_port_get_raw(struct device *port, u32_t *value)
+{
+	const struct gpio_cc32xx_config *gpio_config = DEV_CFG(port);
+	unsigned long port_base = gpio_config->port_base;
+	long status;
+	unsigned char pin_packed = 0xFF;
+
+	status =  MAP_GPIOPinRead(port_base, pin_packed);
```

Comment:
`status` is not the best name in this context. Maybe we should simply write it as
```
	*value =  (u32_t *)MAP_GPIOPinRead(port_base, pin_packed);
```

Reply:
Done